docs/explanations/audit-scanner/audit-scanner.md

```diff
@@ -4,7 +4,8 @@ sidebar_position: 50
 title: What is the Audit Scanner?
 description: An overview of the Kubewarden Audit Scanner.
 keywords: [kubewarden, audit scanner, kubernetes]
-doc-persona: [kubewarden-user, kubewarden-operator, kubewarden-policy-developer, kubewarden-integrator]
+doc-persona:
+  [kubewarden-user, kubewarden-operator, kubewarden-policy-developer, kubewarden-integrator]
```

Comment:
Ok, fine. Is a formatter of some sort being used? We should all maybe be using the same one in that case?